Frontend/src/app/components/research-lab/recency-chart/recency-launch-config/recency-launch-config.component.html

```
@@ -1,81 +1,127 @@
+      <div class="strategy-grid">
+        @for (strategy of eligibleStrategies(); track strategy.name) {
+          @let selected = isSelected(strategy);
+          <article class="strategy-option" [class.strategy-option--selected]="selected">
+            <label>
+              <input type="checkbox" [checked]="selected" (change)="toggleStrategy(strategy)" />
+              <span>{{ strategy.display_name }}</span>
+            </label>
+          </article>
+        }
+      </div>
+      @if (selectedStrategies().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)